src/pages/Product/Product.js

```diff
@@ -13,23 +15,25 @@ class Product extends Component {
   }
 
   componentDidMount() {
-    fetch('/data/ListItemData.json')
+    fetch(`${BASE_URL}/products${this.props.location.search}`)
+      //fetch(`/data/ListItemData.json`)
       .then(data => data.json())
       .then(data => {
-        this.setState({ prolist: data });
+        console.log(data);
+        this.setState({ prolist: data.RESULTS });
       });
   }
 
   render() {
     const { prolist } = this.state;
-    return (
+    return this.state.prolist ? (
```

Comment:
```suggestion
    return this.state.prolist && (
```
여기도 이렇게 써줄 수 있을거 같네요

Reply:
수정완료했습니다!